test/benchee_test.exs

```diff
@@ -253,6 +253,40 @@ defmodule BencheeTest do
     end
   end
 
+  test ".run accepts atom keys for jobs" do
+    capture_io fn ->
```

Comment:
Probably I guess :) 

Plus I have the added challenge that sometimes I do want to capture the IO and do assertions on it so I couldn't just disable it for a whole file :(